<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user</class>
 <widget class="QWidget" name="user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>981</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>志愿者界面</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>571</width>
     <height>851</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="Label">
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="LineEditName">
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>性别</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>男</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>女</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="Label_2">
      <property name="text">
       <string>年龄</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="LineEditAge">
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="Label_3">
      <property name="text">
       <string>电话</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="LineEditTelephone">
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="Label_4">
      <property name="text">
       <string>身份证</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="LineEditIdCard">
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>个人简介</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="Label_5">
      <property name="text">
       <string>是否有志愿者经历</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QCheckBox" name="CheckBox"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>语言掌握情况</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>英语</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string>中文</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QRadioButton" name="radioButton_5">
      <property name="text">
       <string>日语</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QRadioButton" name="radioButton_6">
      <property name="text">
       <string>韩语</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QRadioButton" name="radioButton_7">
      <property name="text">
       <string>印度语</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QRadioButton" name="radioButton_8">
      <property name="text">
       <string>俄语</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="Label_6">
      <property name="text">
       <string>可供的时间</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QLineEdit" name="LineEdit">
      <property name="placeholderText">
       <string>以五对以','分割的时间对表示</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="Label_7">
      <property name="text">
       <string>重置密码</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QLineEdit" name="LineEdit_2">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="Label_8">
      <property name="text">
       <string>重复密码</string>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QLineEdit" name="LineEdit_3">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
   <zorder>radioButton_8</zorder>
   <zorder>Label</zorder>
   <zorder>LineEditName</zorder>
   <zorder>radioButton</zorder>
   <zorder>Label_2</zorder>
   <zorder>LineEditAge</zorder>
   <zorder>Label_3</zorder>
   <zorder>LineEditTelephone</zorder>
   <zorder>Label_4</zorder>
   <zorder>LineEditIdCard</zorder>
   <zorder>label</zorder>
   <zorder>textEdit</zorder>
   <zorder>Label_5</zorder>
   <zorder>CheckBox</zorder>
   <zorder>radioButton_2</zorder>
   <zorder>label_2</zorder>
   <zorder>radioButton_3</zorder>
   <zorder>label_3</zorder>
   <zorder>radioButton_4</zorder>
   <zorder>radioButton_5</zorder>
   <zorder>radioButton_6</zorder>
   <zorder>radioButton_7</zorder>
   <zorder>Label_6</zorder>
   <zorder>LineEdit</zorder>
   <zorder>Label_7</zorder>
   <zorder>LineEdit_2</zorder>
   <zorder>Label_8</zorder>
   <zorder>LineEdit_3</zorder>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>修改个人信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>910</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>910</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出并同步信息到系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>910</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>放弃修改</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
